### ISSUE TITLE
crypto module

### ISSUE THREAD

**Issue by zeke** [joehand/dat-doctor#11]
When installing `dat`, I got a warning about `crypto`:

```
$ mkdir foo && cd foo && npm init --yes
$ npm i dat
npm WARN deprecated crypto@0.0.3: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.

...
+ dat@13.8.1
added 333 packages in 6.364s
```

It appears that `dat-doctor` is using the `crypto` module:

```
$ npm ls crypto
foo@1.0.0 /Users/z/Desktop/foo
└─┬ dat@13.8.1
  └─┬ dat-doctor@1.3.0
    └── crypto@0.0.3 
```

However when I clone the dat-doctor repo and npm install, I don't see `crypto`, even though it's the same version 1.3.0 that `dat` is using...

```
~/git/forks/dat-doctor master
$ npm ls crypto
dat-doctor@1.3.0 /Users/z/git/forks/dat-doctor
└── (empty)
```

🤔

**Comment by derhuerst** [comment]
#10 has fixed this, but apparently this change hasn't been published yet.

**Comment by joehand** [comment]
Published in `v1.3.1`, thanks!

**Comment by zeke** [comment]
👍
